docs/Developers/Development_Setup.rst

```diff
@@ -114,13 +114,19 @@ source in the future.
 Building Documentation
 ----------------------
 
-You should run this command before trying to build the documentation:
+You need to have the required dependencies installed, run this command to
+install the required dependency:
 
 ::
 
     (coala-venv)$ pip3 install -r docs-requirements.txt
 
-Once you have done so, you can build the documentation by entering the docs
-directory and running ``python3 setup.py docs``. The documentation
-on the coala website is in
-the ``coala`` (not ``coala-bears``) repository.
+Once finished, run this command to build the documentation:
+
+::
+
+    (coala-venv)$ python3 setup.py docs
+
+The documentation on the https://docs.coala.io/en/latest/ is in the ``coala``
```

Comment:
unfortunately not. https://docs.coala.io/en/latest/ is in https://github.com/coala/documentation .

https://api.coala.io/en/latest/ is in https://github.com/coala/coala .

Both sites link to each other , and link to github , etc.

This PR needs to talk about both repos/sites , as mentioned at https://github.com/coala/coala/issues/5449#issuecomment-394515378 .  I think you did see that as you mention below that most repos have generated docs, which is good, but the distinction between the two main sites is quite important.

And then there is https://coala.io , which probably used to have some docs on it, but now doesnt, and that is in a different repo.